### PR TITLE
chore: bump hermes to v1.8.0

### PR DESCRIPTION
**Pull request by faddat**
## Description

Works on issue:
* https://github.com/cosmos/cosmos-sdk/issues/12228

This PR updates hermes to v1.7.4 and ensures that hermes is not using a feature branch fork. 

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [x] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] Added `!` to the type prefix if state-machine breaking change (i.e., requires coordinated upgrade)
* [x] Confirmed this PR does not introduce changes requiring state migrations, OR migration code has been added to consumer and/or provider modules
* [x] Targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
* [x] Provided a link to the relevant issue or specification
* [ ] Followed the guidelines for [building SDK modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules)
* [ ] Included the necessary unit and integration [tests](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#testing)
* [ ] Added a changelog entry to `CHANGELOG.md`
* [ ] Included comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] Updated the relevant documentation or specification
* [ ] Reviewed "Files changed" and left comments if necessary 
* [x] Confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] confirmed this PR does not introduce changes requiring state migrations, OR confirmed migration code has been added to consumer and/or provider modules
* [ ] confirmed all author checklist items have been addressed
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -42,9 +42,9 @@ RUN dnf update -y
 RUN dnf install -y which iproute iputils procps-ng vim-minimal tmux net-tools htop jq
 USER root
 
-COPY --from=hermes-builder /usr/bin/hermes /usr/local/bin/
+COPY --from=hermes-builder /usr/bin/hermes /usr/local/bin/hermes
```
Comment (jtremback) [inline Dockerfile:45]: Why did this change?

File: Dockerfile [hunk 2]
```diff
@@ -42,9 +42,9 @@ RUN dnf update -y
...
 COPY --from=cometmock-builder /usr/local/bin/cometmock /usr/local/bin/cometmock
-COPY --from=gorelayer-builder /bin/rly /usr/local/bin/
+COPY --from=gorelayer-builder /bin/rly /usr/local/bin/rly
```
Comment (jtremback) [inline Dockerfile:47]: Why did this change?

**Comment by faddat** [comment]
@jtremback those changes happened because of changes in the docker image used.

File: Dockerfile.gaia [hunk 3]
```diff
@@ -61,14 +61,14 @@ WORKDIR /interchain-security
...
-FROM ghcr.io/informalsystems/hermes:1.4.1 AS hermes-builder
+FROM ghcr.io/informalsystems/hermes:1.5.1 AS hermes-builder
 
 FROM --platform=linux/amd64 fedora:36
 RUN dnf update -y
 RUN dnf install -y which iproute iputils procps-ng vim-minimal tmux net-tools htop jq
 USER root
 
-COPY --from=hermes-builder /usr/bin/hermes /usr/local/bin/
+COPY --from=hermes-builder /usr/bin/hermes /usr/local/bin/hermes
```
[the remark this answers is not in the document]
Comment (faddat) [reply Dockerfile.gaia:71]: docker image changes.  It was necessary to specify the exact path.

File: Dockerfile (hunk 2 above)
Comment (faddat) [reply Dockerfile:47]: docker image changes. It was necessary to specify the exact path.

File: Dockerfile (hunk 1 above)
Comment (faddat) [reply Dockerfile:45]: docker image changes. It was necessary to specify the exact path.

**Comment by faddat** [comment]
@jtremback I'm going to flip this back to a draft, I have to update the configuration used for Hermes because I've now updated this to v1.6.0

**Comment by faddat** [comment]
@jtremback - I think that the remaining issue, may be related to either the new relayer software, or test code, and I also think that it may work fine on a second run.

File: Dockerfile.gaia [hunk 4]
```diff
@@ -61,14 +61,14 @@ WORKDIR /interchain-security
 RUN make install
 
 # Get Hermes build
-FROM ghcr.io/informalsystems/hermes:1.4.1 AS hermes-builder
+FROM ghcr.io/informalsystems/hermes:1.6.0 AS hermes-builder
```
Comment (mpoke) [inline Dockerfile.gaia:64]:
```suggestion
FROM ghcr.io/informalsystems/hermes:1.7.4 AS hermes-builder
```

File: Dockerfile [hunk 5]
```diff
@@ -29,7 +29,7 @@ RUN go mod tidy
 RUN make install
 
 # Get Hermes build
-FROM otacrew/hermes-ics:evidence-cmd AS hermes-builder
+FROM ghcr.io/informalsystems/hermes:1.7.4 AS hermes-builder
```
Comment (sainoe) [inline Dockerfile:32]:
```suggestion
FROM ghcr.io/informalsystems/hermes:v1.7.4 AS hermes-builder
```

**Review by mpoke — APPROVED** [review]
LGTM

**Comment by faddat** [comment]
maybe

**Review by MSalopek — APPROVED** [review]

**Pull request merged**